serverless.yml

```diff
@@ -0,0 +1,224 @@
+# Welcome to Serverless!
```

Comment:
Just small note - I think it would be nice to remove unused/commented properties.